server/sonar-server/src/main/java/org/sonar/server/qualitygate/QualityGates.java

```diff
@@ -133,6 +100,11 @@ public void delete(long idToDelete) {
     }
   }
 
+  /**
+   * Use {@link QualityGateUpdater#setDefault(DbSession, QualityGateDto)}
+   * @deprecated
+   */
+  @Deprecated
   public void setDefault(DbSession dbSession, @Nullable Long idToUseAsDefault) {
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)